devenv/docker/blocks/tempo/tempo.yaml

```diff
@@ -1,5 +1,3 @@
-metrics_generator_enabled: true
```

Comment:
Is this obsolete? Do we generate the metrics some other way? I assume these were generating metrics for the service graph or is this something else?

Reply:
Bottom of this file has `metrics_generator_processors: [service-graphs, span-metrics]` which does the generating.